.ci/validation/test/fixtures/invalid/switch-missing-when.yaml

```diff
@@ -0,0 +1,45 @@
+document:
+  dsl: 1.0.0-alpha2
+  namespace: test
+  name: switch-missing-when
+  version: 0.1.0
+do:
+  - processOrder:
+      switch:
+        - case1:
+            when: .orderType == "electronic"
+            then: processElectronicOrder
+        - case2:
+            when: .orderType == "physical"
+            then: processPhysicalOrder
+        - default:
+            then: handleUnknownOrderType
```

Comment:
```suggestion
          then: handleUnknownOrderType
```

Reply:
this is a intentionally failing example ... see file path